astropy/cosmology/_io/mapping.py

```diff
@@ -80,7 +80,7 @@
 
     >>> cm2 = cm | {"extra": 42, "cosmology": "FlatLambdaCDM"}
     >>> cosmo = Cosmology.from_format(cm2, move_to_meta=True)
-    >>> cosmo.meta
+    >>> cosmo.meta  # doctest: +ELLIPSIS
```

Comment:
Isn't ELLIPSIS turned on by default?

Reply:
Is it? Anyway, I added it hoping it would fix doctest but it didn't. I should probably remove it.